Gemfile

```diff
@@ -1,15 +0,0 @@
-source "https://rubygems.org"
-gem "nokogiri", ">= 1.10.4"
```

Comment:
please leave those files included

Reply:
Thanks a lot for your feedback. I have added those files.